x-pack/plugins/synthetics/public/legacy_uptime/lib/helper/url_params/stringify_url_params.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { stringify } from 'query-string';
+import { UptimeUrlParams } from '..';
+import { CLIENT_DEFAULTS } from '../../../../../common/constants';
+
+const {
+  FOCUS_CONNECTOR_FIELD,
+  DATE_RANGE_START,
+  DATE_RANGE_END,
+  AUTOREFRESH_INTERVAL,
+  AUTOREFRESH_IS_PAUSED,
+} = CLIENT_DEFAULTS;
+
+export const stringifyUrlParams = (params: Partial<UptimeUrlParams>, ignoreEmpty = false) => {
```

Comment:
Is this old but just moved, or is this new logic?

Reply:
it's copied from uptime to have synthetics version.